### PR TITLE
Ab#106800 fixing issue

### PR DESCRIPTION
**Pull request by tc-chris**

### CODE REVIEW

File: src/CSF.SRDashboard.Client/Pages/_Host.cshtml [hunk 1]
```diff
@@ -29,7 +29,14 @@
...
     <link href="~/css/components/component-library.css" rel="stylesheet" />
-    <link href="~/css/site.css" rel="stylesheet" />
+        <link href="~/css/site.css" rel="stylesheet" />
+
+    @*<link rel="stylesheet" href='@Url.Content("CSF.SRDashboard.Client.styles.css")'>
+    <link rel="stylesheet" href='@Url.Content("~/css/components/component-library.css")'>
+    <link rel="stylesheet" href='@Url.Content("~/css/site.css")'>*@
+
+
+
```
Comment (AlexandreSanscartier) [inline src/CSF.SRDashboard.Client/Pages/_Host.cshtml:39]: If we don't need this I'd remove them from the source code.
```suggestion
```

File: src/CSF.SRDashboard.Client/Startup.cs [hunk 2]
```diff
@@ -120,6 +122,17 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env, IKeyVaul
...
+            //{
+            //    // Set the environment on start up
+            //    context.Response.Cookies.Append("Environment", Environment.GetEnvironmentVariable("Environment"), new CookieOptions()
+            //    {
+            //        HttpOnly = true,
+            //    });
+
+            //    return next(context);
+            //});
+
```
Comment (AlexandreSanscartier) [inline src/CSF.SRDashboard.Client/Startup.cs:135]: Remove if no longer needed.
```suggestion
```

File: src/CSF.SRDashboard.Client/appsettings.nca.json [hunk 3]
```diff
@@ -7,8 +7,9 @@
       "Microsoft.Hosting.Lifetime": "Information"
     }
   },
+  "Env": "Acceptance",
   "AzureKeyVaultSettings": {
-    "KeyVaultServiceEndpoint": "https://kv-seafarers-dashboard.vault.azure.net/",
+    "KeyVaultServiceEndpoint": "",
```
Comment (AlexandreSanscartier) [inline src/CSF.SRDashboard.Client/appsettings.nca.json:12]: is there no KeyVaultServiceEndpoing for nca?

File: src/CSF.SRDashboard.Client/appsettings.pcp.json [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
+      "Microsoft": "Warning",
+      "Microsoft.Hosting.Lifetime": "Information"
+    }
+  },
+  "Env": "Production",
+  "AzureKeyVaultSettings": {
+    "KeyVaultServiceEndpoint": "",
```
Comment (AlexandreSanscartier) [inline src/CSF.SRDashboard.Client/appsettings.pcp.json:12]: does PCP have no keyvault endpoints?

**Review by AlexandreSanscartier — COMMENTED** [review]
A few comments, but most of it looks good. I haven't tested building in each environment will give it a go though.

**Review by tropical-turtle — APPROVED** [review]

**Comment by tropical-turtle** [comment]
I strongly suggest adding another app settings file called "appsettings.Development.json" as I mentioned before to make it easier for the developer to run the application.
I know you would say that the developer can pick and choose to run for which environment. Yes, we can pick and choose for the purpose of testing for those environment. But when we are not testing for those different environments, just in Debug configuration, application should be able to pick and choose which app settings to use. I believe this is the essence on this PBI. The rest looks good. Good Job . :)

File: src/CSF.SRDashboard.Client/appsettings.pcp.json (hunk 4 above)
Comment (tc-chris) [reply src/CSF.SRDashboard.Client/appsettings.pcp.json:12]: Is hasn't been created

File: src/CSF.SRDashboard.Client/appsettings.nca.json (hunk 3 above)
Comment (tc-chris) [reply src/CSF.SRDashboard.Client/appsettings.nca.json:12]: Not at the moment

File: src/CSF.SRDashboard.Client/Startup.cs (hunk 2 above)
Comment (tc-chris) [reply src/CSF.SRDashboard.Client/Startup.cs]: Done

**Pull request merged**